features/machine/machine.feature

```diff
@@ -141,4 +141,39 @@ Feature: Machine features testing
     Given I scale the machineset to +2
     Then the machineset should have expected number of running machines
 
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  @destructive
+  Scenario: [MAO] Reconciling machine taints with nodes
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    And I wait for the steps to pass:
+    """
+    Given I clone a machineset named "machineset-24363"
```

Comment:
Step `Given I clone a machineset named "machineset-24363"` doesn't need to be embeded in `I wait for ..`

Reply:
You also need to make sure node number is restored after scenario if you use clone. We have several examples.

Reply:
thanks will remove this

Reply:
After adding "node number is restored .." getting this error -    [09:24:43] INFO> Exit Status: 0
      [09:25:13] ERROR> Test Execution will finish prematurely.
      Failed to restore cluster, expected number of nodes , got 7 (RuntimeError)
      /verification-tests/verification-tests/features/step_definitions/machine.rb:70:in `block (2 levels) in <top (required)>'

Reply:
It worked after adding the step -"Given I store the number of machines in the :num_to_restore clipboard" as suggested by you on slack..
Ran test - 
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/81254/console